### PR TITLE
show ssh-agent failure messages

### PR DESCRIPTION
**Pull request by kanthchandra**
fixes https://github.com/Shippable/cexec/issues/40

Tested this out by running a pre ci boot image which does not have ssh-agent installed in it. and the build failed telling that ssh-agent command not found.

### CODE REVIEW

**Comment by ric03uec** [comment]
@kanthchandra is the `ssh-agent not found` message being dumped on the job console?
i'm not sure how that is being printed since this check is happening in the python code

**Comment by rageshkrishna** [comment]
@ric03uec This will show up in the `wait_for_exit` step where we show the output of `docker logs`. Because this reports an unrecognized exit code, the boot script recognizes it as a boot failure and the group will be displayed in the console.

**Comment by ric03uec** [comment]
got it. i keep forgetting that part :)
@kanthchandra can you rebase this with the latest cexec.

File: execute.py [hunk 1]
```diff
@@ -150,6 +156,12 @@ def _push_coverage_results(self):
         coverage_results['jobId'] = self.job_id
         self.shippable_adapter.post_coverage_results(coverage_results)
 
+    def _check_for_ssh_agent(self):
+        self.log.debug('Inside Execute')
```
Comment (rageshkrishna) [inline execute.py]: Wrong log

**Pull request merged**